<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>measurementFigure</class>
 <widget class="QWidget" name="measurementFigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>9</y>
     <width>1031</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,0,2">
    <item>
     <widget class="QWidget" name="view_elevation" native="true"/>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1">
      <item>
       <widget class="QWidget" name="view_plan" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="view_3d" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
